docs/source/new_language.rst

```diff
@@ -0,0 +1,122 @@
+
+    sudo docker build . -t my/jobeinabox --build-arg TZ="America/Chicago"
+    sudo docker run -d -p 4000:80 --name jobe my/jobeinabox
+
+That will start up your local Jobe container; once this is done, run the tests that you have written::
+
+    sudo docker exec -it jobe /var/www/html/jobe/testsubmit.py
+
+Submit a pull request
++++++++++++++++++++++
+If your tests pass, then submit a pull request to the `Runestone Jobe repo <https://github.com/RunestoneInteractive/jobe>`_ with your updates. Here is an `example Jobe pull request <https://github.com/RunestoneInteractive/jobe/pull/2>`_.
+
+PreTeXt
+-------
+PreTeXt is the markup language used for actually writing the books. When PreTeXt outputs HTML code containing programming code for active use within Runestone, it embeds markup information within to tell the Runestone server what language is being used. You will need to update PreTeXt so that it correctly tags Runestone-viewable code for your language.
+
+The PreTeXt team prefers that you discuss your idea with them on the development email list before you implement it and submit a pull request, so it is recommended that you do that early in the process. Make sure to first check out the `PreTeXt contributing guide <https://github.com/PreTeXtBook/pretext/blob/master/CONTRIBUTING.md>`_. 
+
+Set up a local copy of PreTeXt source code for development
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Make your own personal fork of the `PreTeXt repo <https://github.com/PreTeXtBook/pretext>`_ for making your updates. Then clone it to your own computer.
+
+In order to test your code updates, you will want to be able to run PreTeXt on your own computer. The most straightforward way to do this is to use the `PreTeXt-CLI <https://github.com/PreTeXtBook/pretext-cli?tab=readme-ov-file>`_, which is a separate project for invoking PreTeXt from the command line. Clone that repository. Follow the `instructions in the README <https://github.com/PreTeXtBook/pretext-cli?tab=readme-ov-file>`_ regarding setting up a poetry environment, and activating a shell so that you can run the ``pretext`` script. Then follow the instructions on `using the CLI when doing development on core PreTeXt <https://github.com/PreTeXtBook/pretext-cli/blob/main/docs/core_development.md>`_. 
+
+Once you have your PreTeXt development environment working, test it before making any changes. In particular, start with a Runestone book that has active code within it, and change it to your new language. An easy example is the PreTeXt sample book; look in the file ``rune.xml``, which has a variety of examples. Within the ``<program>`` tag, change the ``language`` property to have your new language. Of course, this won't work yet, but this will help you see what you're looking to test. Once you have made that change, build the book at the command line::
```

Comment:
The preferred example is the runestone sample book as I use that for testing frequently